aurorawatch-pipeline/test_aurora_transform.py

```diff
@@ -0,0 +1,83 @@
+'''Test file for the AuroraWatch extract script.'''
+# pylint: disable=W0212
+
+from unittest.mock import patch
+from datetime import datetime as dt
+
+import pytest
+
+from aurora_transform import get_colour_id, get_current_datetime, transform
+
+
+class TestGetColourID():
```

Comment:
I like the choice of using test classes, tests are clearly groped this way making your code more readable, and they can be run as a group in terminal a lot easier too.